release-team/role-handbooks/docs/README.md

```diff
@@ -63,8 +63,9 @@ General time requirements for leads and shadows are:
 
 In addition to the time requirements above, a Docs Lead must:
 
-- Have the ability to add a milestone to issues, so must be a member of the [milestone maintainers](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/). Access can be requested by creating a [PR](https://github.com/kubernetes/org/pull/2235) against `kubernetes/org` repo.
-- Have the ability to `/approve` PRs. Access can be requested by creating a [PR](https://github.com/kubernetes/website/pull/20351) against `master` branch.
+- Have the ability to add a milestone to issues, so must be a member of the [website milestone maintainers](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/). Access can be requested by creating a [PR](https://github.com/kubernetes/org/pull/2235) against `kubernetes/org` repo.
+> Note: access is restricted to [website milestone maintainers](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/) to members
```

Comment:
Maybe mention that you have to be an org member to even see https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/

Reply:
Thank you for the clarification. Updated the PR